src/renderers/abstract.renderer.ts

```diff
@@ -485,8 +490,33 @@ export abstract class Renderer {
           styleDependenciesList.length !== 0
             ? styleDependenciesList.map((oneCssDep) => genHeaderCSSLink(config, oneCssDep, articleId, config.output.dirs.mediawiki)).join('\n')
             : '',
-        ),
-    )
+        )
```

Comment:
There is a factorisation problem here, this should not be in the abstract class, but in the mobile class.

Reply:
I refactored renderer architecture, there are abstract **MobileRenderer** and abstract **DesktopRenderer** that keep logic for mobile and desktop templating.